_puppeteers/raphink.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: profile
+name: Raphaël Pinson
+image: https://avatars2.githubusercontent.com/u/650430?v=4
```

Comment:
Hmmm, everyone seems to use an actual picture as GitHub avatar... where mine is an octocat...